ui/src/app/settings/containers/Configuration/Configuration.js

```diff
@@ -1,5 +1,31 @@
-import React from "react";
+import { useDispatch, useSelector } from "react-redux";
+import PropTypes from "prop-types";
+import React, { useState, useEffect } from "react";
 
-const Configuration = () => <h4>Configuration</h4>;
+import actions from "app/settings/actions";
+import selectors from "app/settings/selectors";
+import baseSelectors from "app/base/selectors";
+import Loader from "app/base/components/Loader";
+
+const Configuration = () => {
+  const dispatch = useDispatch();
+  const configuration = useSelector(state =>
```

Comment:
rather than just returning the entirety of configuration, it would be nice to have methods which return distinct bits of state.